libraries/AP_GPS/AP_GPS_NOVA.cpp

```diff
@@ -202,10 +202,12 @@ AP_GPS_NOVA::process_message(void)
 
         state.num_sats = bestposu.svsused;
 
-        state.horizontal_accuracy = (float) ((bestposu.latsdev + bestposu.lngsdev)/2);
-        state.vertical_accuracy = (float) bestposu.hgtsdev;
+        state.horizontal_accuracy = (float) ((bestposu.latsdev + bestposu.lngsdev)/2); // RMS
```

Comment:
I'm a bit confused, you have already labeled the `horizontal_accuracy` storage location RMS in it's comments, you updated all the comments for the messages to say the accuracy in millimeters, but omitted the RMS tag there, but then applied it here. This should really just be documented in the header, rather then in line in the code randomly like this, and particularly in the structs that are used to decode the messages.